src/main/java/mate/controller/driver/GetAllDriversController.java

```diff
@@ -3,13 +3,15 @@
 import java.io.IOException;
 import java.util.List;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import mate.lib.Injector;
 import mate.model.Driver;
 import mate.service.DriverService;
 
+@WebServlet(urlPatterns = "/drivers/mycars/")
```

Comment:
I think this is wrong servlet for this annotation
also, if you're using web.xml mapping for everything, let's also map this URL using web.xml